soapui/src/main/java/com/smartbear/swagger/utils/OpenAPIUtils.java

```diff
@@ -0,0 +1,33 @@
+package com.smartbear.swagger.utils;
+
+import com.eviware.soapui.support.xml.XmlUtils;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import io.swagger.util.Json;
+import io.swagger.util.Yaml;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class OpenAPIUtils {
+
+    private static final Logger LOG = LoggerFactory.getLogger(OpenAPIUtils.class);
+
+    public static String extractStringFromExampleObject(String contentType, Object example) {
```

Comment:
This is also part of preparing sample for request and, as I see, it is not used anywhere.